### PR TITLE
Explaining the translation process

### PR DESCRIPTION
**Pull request by danrot**

### CODE REVIEW

**Comment by danrot** [comment]
Looks like something was not merged back to develop from master :smile:

**Comment by wachterjohannes** [comment]
else ACK but someone should review it gramatically (:

**Comment by danrot** [comment]
@chirimoya Can you also check, especially the grammar?

File: book/creating-a-basic-website/adding-a-template.rst [hunk 1]
```diff
@@ -161,6 +161,12 @@ adding tables.
 We've learned how to define the template. The next step is to write 
 corresponding theme.
 
+.. warning::
+
+    The template does not appear in the dropdown of the page form, unless there
```
Comment (dantleech) [inline book/creating-a-basic-website/adding-a-template.rst]: `form unless` no comma

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 2]
```diff
@@ -89,12 +89,33 @@ The excerpt is available in:
...
+The following screenshot shows the `Sulu homepage`_ with the main navigation on
+the right and the footer navigation on the bottom. As you can see the
+navigations returned for the navigation contexts are not necessarily flat, but
+can also contain sub pages.
+
+.. figure:: ../../img/website-navigation-contexts.png
+    :align: center
+
+The navigation contexts can also be used in any other combination you want. The
+separation into main and footer navigation is only a quite common example.
```
Comment (dantleech) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: `a common example` no need for `quite`

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 3]
```diff
@@ -89,12 +89,33 @@ The excerpt is available in:
...
+
+The advantage of this method is that the content manager can decide on his own
```
Comment (dantleech) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: s/his/its

File: book/creating-a-basic-website/setup-a-webspace.rst [hunk 4]
```diff
@@ -114,8 +111,8 @@ the user to add pages to different kind of navigations. The different contexts
 can be defined in the `navigation`-section, and this selection will be
 available to choose from in the administration interface. Afterwards the
 developer can retrieve the navigation for a given context by using some
-Twig-extensions delivered with Sulu.
-
+Twig-extensions delivered with Sulu, whereby it is not only possible to
```
Comment (dantleech) [inline book/creating-a-basic-website/setup-a-webspace.rst]: `Twig extensions included with Sulu`

File: developer/contributing/adding-translations.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
+Adding translations
+===================
+
+For many developer adding translations is the first step in becoming an open
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]: s/developer/developers

File: developer/contributing/adding-translations.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+possible.
+
+Understanding the translation process
+-------------------------------------
+
+First of all it is important to actually understand the translation process of
+Sulu.
+
+Every bundle requiring translations stores them in their
+`Resources/translations/sulu` folder using the xliff format. These files are
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]: Add an external link.

```
using the xliff_ format ...

...
...

.. _xliff: https://en.wikipedia.org/wiki/Xliff
```

(the link must be at the bottom of the document)

File: developer/contributing/adding-translations.rst [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+named after the scheme ``backend.[language].xlf`` and look something like this:
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]: named using the scheme

File: developer/contributing/adding-translations.rst [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+            </body>
+        </file>
+    </xliff>
+
+If the ``SuluTranslateBundle`` is registered in the Kernel, there are two
+commands available to manage Sulu's translations. ``sulu:translate:import``
+takes a language code (e.g. ``en`` or ``de``) and imports all the translation
+files in the given language into the system. Corresponding to that, the command
+``sulu:translate:export`` also takes a language code, and exports the
+translations from all bundles into a single json file located in
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]: s/json/JSON

File: developer/contributing/adding-translations.rst [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+`web/admin/translations`. From there the Sulu-Admin loads the translations
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]: No hyphenation: `Sulu Admin`

File: developer/contributing/adding-translations.rst [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+for the required localization.
+
+Add a new translation
+---------------------
+
+The following guide explains how to add a complete new translation for the
+Sulu-Admin.
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]: No hypenation

File: developer/contributing/adding-translations.rst [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+These steps have to be executed for every single bundle:
+
+#. Copy the file `Resources/translations/sulu/backend.en.xlf` (or any other
+   language you might be more comfortable with) into
+   `Resources/translations/sulu/backend.[language].xlf` (replace
+   ``[language]`` with the 2-letter ISO code of the new language).
```
Comment (dantleech) [inline developer/contributing/adding-translations.rst]:
```
`2 letter ISO code`_

...
...
...

.. _2 letter ISO code: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
```

File: developer/contributing/adding-translations.rst (hunk 9 above)
Comment (danrot) [inline developer/contributing/adding-translations.rst]: We agreed that it is written that way previously.

File: developer/contributing/adding-translations.rst (hunk 11 above)
Comment (danrot) [inline developer/contributing/adding-translations.rst]: I wrote it as `two-letter ISO code`, as it is also done on https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

**Pull request merged**